test/e2e/framework/helpers/k8s/k8s_utils.go

```diff
@@ -442,6 +442,27 @@ func (k *KubeCtl) isPodNotInNS(podName string, namespace string) wait.ConditionF
 	}
 }
 
+// return a condition function that indicates whether the given pod has received the expected event
+func (k *KubeCtl) isPodEventTriggered(namespace string, podName string, expectedReason string) wait.ConditionFunc {
+	return func() (bool, error) {
+		events, err := k.clientSet.CoreV1().Events(namespace).List(context.TODO(), metav1.ListOptions{})
+		if err == nil {
```

Comment:
Nit: revert condition, if `err != nil` and return `false, err`, so you don't need the extra identation.

Reply:
Yes, but we need to duplicate "return false, nil"..right?